### PR TITLE
fix: input classname in affix wrapper

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix missing classname in input when Input has `prefix` or `suffix`          |
| 🇨🇳 Chinese | 修复 Input 有 `prefix` 或者 `suffix` 时 input 缺少某些 className 的问题          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/input/style/mixin.less [hunk 1]
```diff
@@ -7,10 +7,6 @@
 .input-lg() {
   padding: @input-padding-vertical-lg @input-padding-horizontal-lg;
   font-size: @font-size-lg;
-
-  input {
-    font-size: @font-size-lg;
-  }
```
Comment (afc163) [inline components/input/style/mixin.less:13]: 需要发 patch 么

Comment (MadCcc) [reply components/input/style/mixin.less:13]: 能发的话是最好了。。上周发现的问题没蹭上车

Comment (MadCcc) [reply components/input/style/mixin.less:13]: 这边删掉是因为 cls 透传下去之后就不需要了，原本的实现

**Comment by afc163** [comment]
图片

Changelog 里最好写用户问题，不要写修复方式。

**Pull request merged**